mopidy/http/actor.py

```diff
@@ -167,7 +167,11 @@ def _get_static_request_handlers(self):
         return result
 
     def _get_mopidy_request_handlers(self):
+        apps = [app['name'] for app in self.apps]
+        default_webapp = self.config['http']['default_webapp']
+        default_webapp = default_webapp if default_webapp in apps else 'mopidy'
```

Comment:
Do you think it's worth logging a warning message if `default webapp` is set but it's not in `apps` (or equal to 'mopidy')?

Reply:
I don't think we should log if default_app is set to its default value ('mopidy'), but i'll log if the app can't be found yes! 

About the test, I tried few days ago, and could'nt figure how to do it, i'll rework on it